### ISSUE TITLE
Textbox: placeholder text fails color contrast requirements

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#1478]
Design team are aware of this issue and are working on a fix.

**Comment by rbascue** [comment]
We will be updating grey5 from `#767676` to `#707070`. The placeholder text should be using this alias and should pass after being updated.

**Comment by rbascue** [comment]
Created another ticket for updating grey5 #1479
